kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/ConsumerSeekUtil.java

```diff
@@ -54,7 +53,22 @@ private ConsumerSeekUtil() {
      */
     public static void seekToCurrentPositions(Consumer<?, ?> consumer, Supplier<KafkaTrackingToken> tokenSupplier,
                                               List<String> topics) {
-        List<TopicPartition> all = topicPartitions(consumer, topics);
+        seekToCurrentPositions(consumer, tokenSupplier, new TopicListSubscriber(topics));
+    }
+
+    /**
+     * Assigns the correct {@link TopicPartition partitions} to the consumer, and seeks to the correct offset, using the
+     * {@link KafkaTrackingToken}, defaulting to the head of the partition. So for each {@link TopicPartition partition}
+     * that belongs to the {@code topics}, either it will start reading from the next record of the partition, if
```

Comment:
Nit: `topics` is no longer a provided parameter. I would refer to the `subscriber` instead.